docs/src/image_loading.md

```diff
@@ -0,0 +1,143 @@
+# Image Loading
+
+## Overview
+
+The Image Loading module is a component of the MCU Runtime SDK designed for managing SOC images. This module provides APIs for:
+
+- Loading SOC images to target components. The SOC images could come from a [Flash Storage](./flash_layout.md) or from another platform (e.g. BMC).
+- Retrieving the SOC Image Metadata (as defined in the [SOC Manifest](https://github.com/chipsalliance/caliptra-sw/blob/main-2.x/auth-manifest/README.md) documentation)
+- Verifying and authenticating the SOC Images.
```

Comment:
Image loading module won't authenticate SOC image directly. It interacts with Caliptra RT via mailbox interface to achieve the function. 
Can we clarify this?

Reply:
Yeah, I can probably reword this to mean that it provides the interface for authentication through Caliptra core but not itself doing the authentication.